src/pages/hooks/StateHookPage.js

```diff
@@ -29,10 +38,11 @@ export const StateHookPage = () => {
           <button type={'submit'}>Greet me!</button>
         </form>
         <h2 className="inverted">Exercise result</h2>
-        <p>
-          {/* Get greeting according the time */ 'Hi'},{' '}
-          {/* Name goes here */ 'so-and-so'}!
-        </p>
+        {click ? (
```

Comment:
You can also replace the condition `click` with `username && currentTime` to improve your code.
Good job anyway! 😉

Reply:
Done!